mapviz/src/mapviz.cpp

```diff
@@ -924,35 +895,32 @@ void Mapviz::SaveConfig()
 
 void Mapviz::ClearHistory()
 {
-  ROS_DEBUG("Mapviz::ClearHistory()");
-  for (auto& plugin: plugins_)
-  {
-    plugin.second->ClearHistory();  
+  RCLCPP_DEBUG(node_->get_logger(), "Mapviz::ClearHistory()");
+  for (auto& plugin : plugins_) {
+    plugin.second->ClearHistory();
   }
 }
 
 void Mapviz::SelectNewDisplay()
 {
-  ROS_INFO("Select new display ...");
+  RCLCPP_INFO(rclcpp::get_logger("mapviz"), "Select new display ...");
```

Comment:
I have multiple mixed feelings about this.  One is that I just generally don't like macros because they break namespace scoping and IDE have trouble parsing them.  It'd be nice to not have to type quite so much, but this isn't always what you want; most of the time you'd want to pass in `node_->get_logger()` as this argument, but sometimes we don't have a node available and want to create a default logger (as in this case), and in other cases we want to use a different logger entirely.  In fact, it'd probably be better to have plugins make their won sub-loggers instead of all using the same one.  Making our own macro also makes things less readable for anybody who's used to the official ROS way of doing things but isn't familiar with mapviz.

I'm not sure what a great solution to all that is, so in the end I fall back to just doing a little extra typing and using RCLCPP_INFO again...